Client/src/components/app-expandible/AppExpandible.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    ref="expandible"
+    class="expandible"
+    :class="{ 'is-expanded': isExpanded }"
+  >
+    <button
+      aria-expanded="false"
+      class="expandible-button"
+      @click="isExpanded = !isExpanded"
+    >
+      <slot name="buttonText" />
+    </button>
+    <div
+      v-if="isExpanded"
+      class="expandible-panel"
+    >
+      <slot name="content" />
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { ref } from 'vue';
+import { useClickOutside } from './../../composables/useClickOutside';
+
+const expandible = ref(null);
+let isExpanded = ref(false);
```

Comment:
Waarom is de ene `ref` als `const` en de andere als `let`? Voor zover ik begrijp kan het beide in een `const` omdat je niet het type data veranderd maar de value update. Ik ben benieuwd ;-)